README.md

```diff
@@ -1,9 +1,9 @@
-rust-GSL [![Build Status](https://api.travis-ci.org/GuillaumeGomez/rust-GSL.png?branch=master)](https://travis-ci.org/GuillaumeGomez/rust-GSL) [![Gitter](https://badges.gitter.im/Join Chat.svg)](https://gitter.im/GuillaumeGomez/rust-GSL?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+rust-GSL [![Build Status](https://api.travis-ci.org/GuillaumeGomez/rust-GSL.png?branch=master)](https://travis-ci.org/GuillaumeGomez/rust-GSL) [![Gitter](https://badges.gitter.im/JoinChat.svg)](https://gitter.im/GuillaumeGomez/rust-GSL?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 ========
 
-A __Rust__ binding for the [GSL library][] (the GNU Scientific Library).
+A __Rust__ binding for the [GSL library] (the GNU Scientific Library).
```

Comment:
Are you sure this will work? Because I'm not sure if the whitespace will remove the automatic link to the `()` part or not.